webapp/channels/src/sass/components/_post.scss

```diff
@@ -862,7 +862,7 @@
         }
 
         .post__body {
-            padding: 2px 0;
```

Comment:
In the commit that introduced the issue, this line had a transformation issue https://github.com/mattermost/mattermost/pull/31008/files#diff-0c8c23a05f54120c25e079c03c35499267fc0667b6bd4711add62d4e690b9650L857

so, this: `padding: 2px 0 0;` was transformed to this: `padding: 2px 0;` but even when was not sematically written correctly, the original version was interpreted by smart browsers as `padding: 2px 0 0 0;`, but the new version was interpreted as `padding: 2px 0 2px 0`; causing the dynamic virtualized list issue.